### PR TITLE
App string suggestions

### PR DESCRIPTION
**Pull request by comradekingu**
Don't know how these Dart files work, but these are some fixes anyhow.
There is still some title casing where I am not sure if there should be or not.

### CODE REVIEW

**Comment by Clashkid155** [comment]
Nice work however some modifications aren't necessary.
[L165](https://github.com/SongTube/SongTube-App/blob/672c7c6d0ea5da7f957db13a307aee21d2913fd0/lib/internal/languages/languageEn.dart#L165): By doing this you caused inconsistent in spelling considering color is also a valid word.
[L213](https://github.com/SongTube/SongTube-App/blob/672c7c6d0ea5da7f957db13a307aee21d2913fd0/lib/internal/languages/languageEn.dart#L213): Shouldn't be changed.
[L249](https://github.com/SongTube/SongTube-App/blob/672c7c6d0ea5da7f957db13a307aee21d2913fd0/lib/internal/languages/languageEn.dart#L249): Space missing between using and Android.
[L179](https://github.com/SongTube/SongTube-App/blob/672c7c6d0ea5da7f957db13a307aee21d2913fd0/lib/internal/languages/languageEn.dart#L179): Shouldn't be changed.

File: lib/internal/languages/languageEn.dart [hunk 1]
```diff
@@ -162,69 +162,69 @@ class LanguageEn extends Languages {
   @override
   String get labelLicenses => "Licenses";
   @override
-  String get labelChooseColor => "Choose Color";
+  String get labelChooseColor => "Choose Colour";
```
Comment (comradekingu) [inline lib/internal/languages/languageEn.dart:165]:
```suggestion
  String get labelChooseColor => "Choose Color";
```

File: lib/internal/languages/languageEn.dart [hunk 2]
```diff
@@ -162,69 +162,69 @@ class LanguageEn extends Languages {
...
   @override
-  String get labelAndroid11FixJustification => "Fixes Download issues on " +
-    "Android 10 & 11";
+  String get labelAndroid11FixJustification => "Fixes downloading issues on " +
+    "Android 10 and 11";
   @override
   String get labelBackup => "Backup";
   @override
-  String get labelBackupJustification => "Backup your media library";
+  String get labelBackupJustification => "Back up your media library";
```
Comment (comradekingu) [inline lib/internal/languages/languageEn.dart:213]: @Clashkid155 If this shouldn't be changed. Is it actually "Backup of your media library"? "Back up *" is a verb. "Backup" is a noun.

File: lib/internal/languages/languageEn.dart [hunk 3]
```diff
@@ -246,12 +246,11 @@ class LanguageEn extends Languages {
 
   // Android 10 or 11 Detected Dialog
   @override
-  String get labelAndroid11Detected => "Android 10 or 11 Detected";
+  String get labelAndroid11Detected => "You are usingAndroid 10 or 11";
```
Comment (comradekingu) [inline lib/internal/languages/languageEn.dart:249]:
```suggestion
  String get labelAndroid11Detected => "You are using Android 10 or 11";
```

File: lib/internal/languages/languageEn.dart [hunk 4]
```diff
@@ -162,69 +162,69 @@ class LanguageEn extends Languages {
...
   @override
-  String get labelUseSystemTheme => "Use System Theme";
+  String get labelUseSystemTheme => "System";
   @override
   String get labelUseSystemThemeJustification =>
-    "Enable/Disable automatic Theme";
+    "Automatic Theme";
   @override
-  String get labelEnableDarkTheme => "Enable Dark Theme";
+  String get labelEnableDarkTheme => "Dark";
```
Comment (comradekingu) [inline lib/internal/languages/languageEn.dart:174]: @Clashkid155 If 179 should be reverted, why is "Dark" OK?
They are both under "Theme".

Comment (Clashkid155) [reply lib/internal/languages/languageEn.dart:174]: Wasn't supposed to include this forget about it.

File: lib/internal/languages/languageEn.dart (hunk 2 above)
Comment (Clashkid155) [reply lib/internal/languages/languageEn.dart:213]: I see what you mean, carry on.

**Comment by comradekingu** [comment]
@Clashkid155 Good?

**Comment by Clashkid155** [comment]
Nice but it has conflicts with the main branch

**Pull request closed without merging**